### PR TITLE
Polish designer job description

### PR DESCRIPTION
**Pull request by mbrookes**

### CODE REVIEW

File: docs/src/pages/company/careers/designer.md [hunk 1]
```diff
@@ -1,101 +1,101 @@
...
+We've seen rapid growth with all of them (2-3x per year) and we have more to come.
 
-We are a team of fully distributed individuals, spread across multiple timezones.
-We work mainly asynchronously, relying on lots of documentation and written collaboration.
+We are a fully distributed team, spread across multiple timezones.
+We work mainly asynchronously, relying mostly on written collaboration.
 Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
-For more details about MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+For additional details about the MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://www.notion.so/Handbook-f086d47e10794d5e839aef9dc67f324b).
```
Comment (oliviertassinari) [inline docs/src/pages/company/careers/designer.md]:
```suggestion
For additional details about the MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
```

File: docs/src/pages/company/careers/designer.md [hunk 2]
```diff
@@ -1,101 +1,101 @@
...
 
-We offer competitive compensation depending on the profile and location.
+We offer competitive compensation, aligned with your profile and location.
 We're ready to pay top market rates for a designer that can significantly push the mission forward.
 Other perks are described on [the careers page](/careers/#perks-amp-benefits/).
 
 ## How to apply?
 
-[Apply now for this position 📮](https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying+for=Designer).
+[Apply now for this position 📮](https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying for=Designer).
```
Comment (oliviertassinari) [inline docs/src/pages/company/careers/designer.md:82]:
```suggestion
[Apply now for this position 📮](https://airtable.com/shrdqo1Z6srZXGcvh?prefill_Applying+for=Designer).
```

File: docs/src/pages/company/careers/designer.md [hunk 3]
```diff
@@ -1,101 +1,101 @@
...
+MUI started with Material-UI, the most successful React implementation of Google's Material Design.
+It has gained a large following, not only due to the fidelity to Material Design, but also because of the number of components, its carefully designed component API, obsession for details, and community engagement.
 Today, countless teams and organizations rely on our open-source projects to build their design system.
 
-A few years back, we started expanding our suite of products.
-We released [MUI X](/x/), a collection of advanced components, [MUI Design kits](/design-kits/), MUI components available in the most popular design tools out there, and [MUI Templates](/templates/), a set of pre-built UI.
-We've seen fast growth in all of them (x2 ~ 3 year over year) and we have much more baking in the oven.
+A couple of years ago, we started to expand our suite of products.
+We released [MUI X](/x/), a collection of advanced components; [MUI Design Kits](/design-kits/), the MUI Material components available for the most popular design tools; and also host [Templates](/templates/), a set of pre-built UI kits.
```
Comment (oliviertassinari) [inline docs/src/pages/company/careers/designer.md]:
```suggestion
We released [MUI X](/x/), a collection of advanced components; [MUI Design kits](/design-kits/), the MUI components available for the most popular design tools; and also host [Templates](/templates/), a set of pre-built UI kits.
```

File: docs/src/pages/company/careers/designer.md (hunk 2 above)
Comment (mbrookes) [reply docs/src/pages/company/careers/designer.md:82]: 👍  @oliviertassinari – Atrefact of the urldecode of the version rescued  from request headers :D

**Pull request closed without merging**